### PR TITLE
Feature/rimworld v1.1

### PR DESCRIPTION
**Pull request by rshackleton**
Copied the changes made to the main Locks mod based on this PR:

https://github.com/Aviuz/Locks/pull/21

Tested with a pretty heavy modlist and locking/unlocking the new doors worked as expected.

### CODE REVIEW

**Comment by rshackleton** [comment]
Tried with the Prison Labour mod and prisoners aren't pathing through the unlocked Doors Expanded doors, but they are pathing through unlocked vanilla doors.

**Comment by rshackleton** [comment]
Fixed, needed to update the harmony patch to target `Building_DoorExpanded`

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Changelog:
+2.4.0
```
Comment (Hazzer) [inline changelog.txt:2]: Why version 2.4.0?

**Comment by Hazzer** [comment]
Thanks for update!

File: changelog.txt (hunk 1 above)
Comment (rshackleton) [reply changelog.txt:2]: I have no idea! I assume I meant to type 2.3.0.

**Pull request merged**